test/test_mkldnn.py

```diff
@@ -105,7 +106,7 @@ def test_conv2d(self):
             N = torch.randint(3, 10, (1,)).item()
             C = torch.randint(1, 3, (1,)).item() * groups
             M = torch.randint(1, 3, (1,)).item() * groups
-            x = torch.randn(N, C, 224, 224, dtype=torch.float32) * 100
+            x = torch.randn(N, C, 224, 224, dtype=torch.float32)
```

Comment:
Why removing `* 100`?

Reply:
There's an issue is this test script: it didn't check conv results between mkldnn and aten implementation, instead, it basically compared the mkldnn with mkldnn, because default cpu path will also go into the mkldnn. Now I used this flag added by Jiayi to fix the issue
```python
with torch.backends.mkldnn.flags(enabled=False):
```

As a result, the mkldnn output will diverse with aten output if we scale up the input tensor. Also, `assertEqual` doesn't have the parameter like `rtol` in `torch.allclose` for tuning the relative error, however, it only has a parameter controlling the absolute error (1e-5 by default).
So there's three options now: 1) don't enlarge the input, 2) use allclose instead, 3) change absolute error to 1e-2 or 1e-3. And I took the first for now.

Reply:
OK. Option 1 makes sense considering only absolute error is checked by `assertEqual`.